packages/android_alarm_manager/android/src/main/java/io/flutter/plugins/androidalarmmanager/AndroidAlarmManagerPlugin.java

```diff
@@ -41,7 +48,12 @@ public void onMethodCall(MethodCall call, Result result) {
     String method = call.method;
     Object arguments = call.arguments;
     try {
-      if (method.equals("Alarm.periodic")) {
+      if (method.equals("AlarmService.start")) {
+        startService((JSONArray) arguments);
+        result.success(true);
+      } else if (method.equals("AlarmService.initialized")) {
+        AlarmService.onInitialized();
```

Comment:
@bkonyi I think we're missing the invocation of the result callback?

Reply:
Yeah, looks like it. I don't think it causes any problems though, does it? I don't believe the background isolate cares if it gets a response here.

Reply:
I'm just going through the fun task of auditing all of our invokeMethod calls as part of reviewing #1365 and was confused when looking at this.

I'm guessing it will result in the future here never completing:
https://github.com/flutter/flutter/blob/ecfdd7e1ea96be3c762af6a1bd6be63d9e4d9682/packages/flutter/lib/src/services/platform_channel.dart#L295 

Probably best to clean it up anyway.